### PR TITLE
new article: lets play with nn(1)

### PR DESCRIPTION
**Pull request by ryanchao2012**

### CODE REVIEW

**Comment by ChuyuHsu** [comment]
What does this file `blog/test.xml` for?
Can you remove it?

**Comment by ChuyuHsu** [comment]
I saw you using the reference like citation
Update your references by the following style

Markdown:

```
I bet you'd like more information about this sentence [^1].

[^1]: Well lucky for you, I've included more information in footnote form.
```

File: _posts/2015-12-07-nn-basic.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+$m$: number of the samples in one "batch" we feed into NN.
+$K$: number of labels in multi-classification problem.
+$(x^{(i)}, y^{(i)})$: $(feature, \ label)$ pair of sample $i$.
+$L$: total number of $\Theta$, which is, the matrices used to do mapping from layer to layer.
+$\lambda$: regularization parameter.
+$\delta_{j}^{(L)}$: the "error" at node $j$ in the output layer.
+$\delta_{j}^{(l)}$: the algorithm loops the equation(2) and (3) to  compute the "error" backward in every hidden layer $l$ and get the partial derivative at every weight $\frac{\partial J}{\partial \Theta_{ij}}$.
+$a_{j}^{(l)}$: the activation value at node $j$ in hidden layer $l$.
+$.*$: symbol of element-wise multiplication.[4] 
+Yut can find more mathematical details from the reference[3].
```
Comment (ryanchao2012) [inline _posts/2015-12-07-nn-basic.md]: typo: Yut --> You

**Comment by mayyangtaiwan** [comment]
Done my review. All good. Someone please review the formula/code part in details.

**Comment by mayyangtaiwan** [comment]
Done my review. All good. Someone please review the formula/code part in details.

**Comment by ChuyuHsu** [comment]
@ryanchao2012  英文大致沒問題, 
麻煩將 `blog/test.xml`移除, 
將reference改成footnote
還有你自己的typo就可以merge了

**Pull request merged**